README.md

```diff
@@ -25,6 +25,10 @@ The official [AKS Engine documentation][docs] covers everything you need to know
 
 Want to get involved? The [community guide][community] covers everything you need to know about the AKS Engine community and how you can contribute.
 
+## Support
+
+AKS Engine is an open source project that is **not** backed by a Microsoft Azure 24/7 service support model. [Please search open issues here](https://github.com/Azure/aks-engine/issues), and [open a new one](https://github.com/Azure/aks-engine/issues/new/choose) if your scenario isn't already represented in an existing issue, and project maintainers will engage in a timely fashion!
```

Comment:
I suggest
"AKS Engine is an open source project that is **not** covered by the [Microsoft Azure support policy](https://support.microsoft.com/en-us/help/2941892/support-for-linux-and-open-source-technology-in-azure). [Please search open issues here](https://github.com/Azure/aks-engine/issues), and if your issue isn't already represented please [open a new one](https://github.com/Azure/aks-engine/issues/new/choose). The aks-engine project maintainers will respond to the best of their ability."

I propose that we also start two slack channels: 1) #aks-engine-users and 2) #aks-engine-dev and start holding public meetings. Once those are created we can list them here for joining the community.

Reply:
I added a task for the slack channels to our [backlog project](https://github.com/Azure/aks-engine/projects/2).